docs/infrastructure/microk8s-quickstart.md

```diff
@@ -80,18 +77,16 @@ Now that MicroK8s is running on the machine we can proceed to enabling the neces
     microk8s.status --wait-ready
     ```
 
-4.  Install version v2.16.3 of the Helm client:
+4.  Export your kubeconfig so that Helm knows on which cluster to install the charts:
 
     ```bash
-    sudo snap install helm --classic --channel=2.16/stable
+    microk8s.config > ~/.kube/config
     ```
 
-5.  Install the Helm server:
+5.  Install Helm vervsion 3.2.0:
```

Comment:
I think we ended up deciding that only the major and minor version numbers were enough. Also, fix a typo:

```suggestion
5.  Install Helm version 3.2:
```